osu.Game/Modes/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -85,6 +84,19 @@ protected override void LoadComplete()
             Expire(true);
         }
 
+        private List<DrawableHitObject> nestedHitObjects;

```

Comment:
Because this isn't just for sliders. For the time being as sliders have the custom positioning logic I would rather not implement this here just yet.

Also gotta consider potential draw depth issues.